components/ReportFilter/index.js

```diff
@@ -79,9 +80,7 @@ export default function ReportFilter({ filter, setFilter, categories }) {
           />
         </Styled.FilterItem>
         <Styled.FilterItem>
-          <Styled.FilterButton onClick={handleResetFilters}>
-            Clear Filter
-          </Styled.FilterButton>
+          <GlobalButton onClick={handleResetFilters} name="Clear Filter" />
```

Comment:
When you use the Styled Component directly instead of your wrapper component you can pass down children directly: 
```jsx
<Styled.Button onClick={...}>Clear Filter</Styled.Button>
```